githubdelivery/src/main/java/io/dockstore/githubdelivery/GithubDeliveryConfig.java

```diff
@@ -0,0 +1,35 @@
+package io.dockstore.githubdelivery;
+
+import io.dockstore.utils.ConfigFileUtils;
+import org.apache.commons.configuration2.INIConfiguration;
+import org.apache.commons.configuration2.SubnodeConfiguration;
+
+public class GithubDeliveryConfig {
+    private DockstoreConfig dockstoreConfig;
+    private S3Config s3Config;
+    public GithubDeliveryConfig(INIConfiguration config) {
+        SubnodeConfiguration dockstoreSection = ConfigFileUtils.getDockstoreSection(config);
+        SubnodeConfiguration s3Section = config.getSection("s3");
+
+        this.dockstoreConfig = new DockstoreConfig(dockstoreSection.getString("server-url", "http://localhost:8080"), dockstoreSection.getString("token"));
+        this.s3Config = new S3Config(s3Section.getString("bucketName", "local-github-delivery-bucket"), s3Section.getString("endpointOverride"));
```

Comment:
I don't think you need the endpoint override unless you plan on writing localstack tests for this module